ci/jenkins_status.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+
+"""Parse Jenkins build test results"""
+
+import argparse
+import json
+import sys
+import urllib
+from urllib.request import urlopen
+
+JENKINS_HOME = "https://build.hpdd.intel.com/job/daos-stack"
+
+
+class TestResult:
+    """Represents a single Jenkins test result"""
+
+    def __init__(self, data, blocks):
+        name = data["name"]
+        if "-" in name:
+            try:
+                (num, full) = name.split("-", 1)
+                test_num = int(num)
+                if full[-5] == "-":
+                    full = full[:-5]
+                name = f"{full} ({test_num})"
```

Comment:
What does the original `data["name"]` look like?

Reply:
this is the weird 8 digits of sha hash at the end of the PR name which is making matching fail.